Makefile

```diff
@@ -100,17 +90,20 @@ all: dependencies lint check-license-header unit integration e2e-compile elastic
 dependencies:
 	go mod tidy -v && go mod download
 
-# Generate code
+# Generate code, CRDs and documentation
+ALL_IN_ONE_CRDS=config/crds-patches/all-in-one.yaml
```

Comment:
What do you think of changing this to `config/crds-patches/all-in-one.yaml/all-in-one-crds.yaml`? At least when I was reading this the first time through I got confused between this and `config/all-in-one.yaml` which also includes the operator resources.

Reply:
`all-in-one-crds.yaml` or `k8s.elastic.co_all-crds.yaml` to stay close to the naming of the crds.

Reply:
Small reorganization proposal:

```sh
> tree config/crds 
config/crds
├── all-crds.yaml
├── bases
│   ├── apm.k8s.elastic.co_apmservers.yaml
│   ├── elasticsearch.k8s.elastic.co_elasticsearches.yaml
│   ├── kibana.k8s.elastic.co_kibanas.yaml
│   └── kustomization.yaml
└── patches
    ├── apm-and-kibana-podtemplate-patch.yaml
    ├── elasticsearch-podtemplate-patch.yaml
    └── kustomization.yaml
```

Advantage: 
- everything related to the CRDs is under `config/crds`
- make a better separation between the crds bases, the crds patches and the final generated file `all-crds.yml`

Reply:
looks good @thbkrkr, let's do that!